crates/runtime_crypto/Cargo.toml

```
@@ -17,5 +17,8 @@ aes = "0.8.2"
 aes-gcm = "0.10.1"
 cbc = { version = "0.1.2", features = ["std"] }
 ring = { version = "0.16.20", features = ["std"] }
+once_cell = "1.10.0"
+num-traits = "0.2.14"
+rsa = { version = "=0.7.0-pre", default-features = false, features = ["std"] }
```

Could we use the latest version of these crates?

```suggestion
once_cell = "1.17.1"
num-traits = "0.2.15"
rsa = { version = "=0.9.2", default-features = false, features = ["std"] }
```

---

yes, we can use the latest version.